jaeger-spark-dependencies/src/main/java/io/jaegertracing/spark/dependencies/DependenciesSparkJob.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2017 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.spark.dependencies;
+
+import io.jaegertracing.spark.dependencies.cassandra.CassandraDependenciesJob;
+import io.jaegertracing.spark.dependencies.elastic.ElasticsearchDependenciesJob;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.TimeZone;
+
+public final class DependenciesSparkJob {
+
+  public static void main(String[] args) throws UnsupportedEncodingException {
+    if (args.length != 1) {
+      throw new IllegalArgumentException("");
+    }
+
+    String storage = System.getenv("STORAGE");
+    if (storage == null) {
+      throw new IllegalArgumentException("Missing environmental variable STORAGE");
```

Comment:
At the this level, it's better to just print out the message and exit with `System.exit(rc)` than throwing an exception.